packages/lib/gpcircuits/circuits/uniqueness.circom

```diff
@@ -19,26 +19,26 @@ template UniquenessModule(
     signal output valuesAreUnique;
 
     // Array of field elements indicating whether the corresponding
-    // element of `values` is unique, i.e. isUnique[i] = 0 iff
-    // values[i] is not unique.
-    signal isUnique[NUM_LIST_ELEMENTS];
+    // element of `values` has no duplicates following it in `values`,
+    // i.e. noDupsAfter[i] = 0 iff this is the case.
```

Comment:
Shouldn't this be `!= 0` rather than `= 0`?  Otherwise I'd reverse the naming of the variable so it indicates what's true, not what's false.